vfile-reporter-folder-json.js

```diff
@@ -1,9 +1,10 @@
 const { sep } = require("path");
 
 class vFolder {
-  constructor({ name = "", children = [] } = {}) {
+  constructor({ name = "", children = [], parentPath = "" } = {}) {
```

Comment:
Could `parentPath` be considered a working directory in this case?
perhaps `cwd` as the variable name?